spec/features/todo_list_item_spec.rb

```diff
@@ -2,28 +2,80 @@
 
 feature "Todo list item", :type => :feature do
 
-  scenario "create to-do item" do
+  scenario "create item" do
+
+    # user var equates to signing in a user with said credentials.
     user = sign_in("tester@example.tld", "test-password")
+
+    # create a list which is for above user, with the title 'to-do list'
     create(:list, user: user, title: "to-do list")
 
+    # visit /lists and click the link (list) 'to-do list'
     visit lists_path
     click_link "to-do list"
 
+    # create list var that equates to the last list element (which will be
+    # the above list we just created)
     list = List.last
+
+    # current path should now be /list/:id, passing in the above list var
     expect(current_path).to eq(list_path(list))
 
+    # page should have the list 'to-do list' on it
     expect(page).to have_content("to-do list")
+
+    # click create new to-do
     click_link "Create new to-do"
 
+    # takes you to new item form, controller#action pair of: items#new
     expect(page).to have_content("Create a new to-do item")
+
+    # page has the content 'create...'
     fill_in "Name", with: "purchase groceries"
+
+    # fill out form and click button 'create new to-do'
     click_button "Create new to-do"
 
+    # this redirects and current path should be /lists/:id
     expect(current_path).to eq(list_path(list))
+
+    # page to have content 'your to-do was created'
     expect(page).to have_content("Your to-do was created.")
+
+    # and also have the new to-do item, DONE!!
     expect(page).to have_content("purchase groceries")
   end
 
```

Comment:
Consider for a future pull request, adding a spec for specifying the behaviour of what happens when a user tries to create a todo with a blank name.